zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -1263,6 +1282,11 @@ function ParagraphCtrl($scope, $rootScope, $route, $window, $routeParams, $locat
       $route.current.pathParams.noteId);
   };
 
+  const patchParagraph = function(paragraph, patch) {
+    const id = paragraph.id;
```

Comment:
It looks unnecessary as `id` is used only once. and this function is only used inside `sendPatch`. Isn't it ok to merge this function into that?